pkg/gce-pd-csi-driver/node.go

```diff
@@ -342,10 +342,12 @@ func (ns *GCENodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 
 	// Part 4: Resize filesystem.
 	// https://github.com/kubernetes/kubernetes/issues/94929
-	resizer := resizefs.NewResizeFs(ns.Mounter)
-	_, err = ns.DeviceUtils.Resize(resizer, devicePath, stagingTargetPath)
-	if err != nil {
-		return nil, status.Error(codes.Internal, fmt.Sprintf("error when resizing volume %s from device '%s' at path '%s': %v", volumeID, devicePath, stagingTargetPath, err.Error()))
+	if !readonly {
```

Comment:
This commit is named "Add tests..." and is changing non-test code. This should be in a separate commit.

Reply:
Again, this will merge with squash so individual commits will go away. Are you suggesting moving this into a separate PR?

Reply:
Sure, if we're squashing then commit separation does not affect the history - so it does not help here. But where it helps a lot are reviews. If commits are not well organized (and described) then reviewers have to look at the PR diff as a whole and it can get difficult to read it. It's not a hard requirement for sure, just a suggestion.

Reply:
Gotcha. For a commit this small I'd argue it's more hassle than benefit to go back and clean up commits when they will be squashed, but I'll definitely keep that in mind in the future as I didn't realize folks look at individual commits during review. I've only ever looked at the entire diff. Thanks for bringing it up though so I know in the future!